client/plots/singleCellPlot.js

```diff
@@ -95,13 +95,15 @@ class singleCellPlot {
 			label: 'Plots',
 			id: PLOTS_TAB,
 			active: activeTab == PLOTS_TAB,
+			isVisible: () => initialSample || this.state?.config.sample,
 			callback: () => this.setActiveTab(PLOTS_TAB)
 		})
 		if (state.termdbConfig.queries?.singleCell?.DEgenes) {
 			this.tabs.push({
 				label: 'Differential Expression',
 				id: DIFFERENTIAL_EXPRESSION_TAB,
 				active: activeTab == DIFFERENTIAL_EXPRESSION_TAB,
+				isVisible: () => initialSample || this.state?.config.sample,
```

Comment:
I think this should be `state.config.sample !== undefined`, and not have to use an `initialSample` variable.

Reply:
Solved now